notebooks/test_output/QC/Tellseq_plate_df_C_set_col19to24.txt

```diff
@@ -1,7 +1,7 @@
 Sample	Col	Compressed Plate Name	Date	Plate Position	Project Abbreviation	Project Name	Project Plate	RackID	Row	Time	TubeCode	Well	vol_extracted_elution_ul	well_id_96	LocationCell	LocationColumn	LocationRow	description	Blank	Sample DNA Concentration_10to1dilution	Sample DNA Concentration_2to1dilution	Sample DNA Concentration_undiluted	Sample DNA Concentration	Diluted	dilution_factor_10to1dilution	Library Well	contains_replicates	Normalized DNA volume	Normalized water volume	Input DNA	syndna_pool_number	Library Well_row	Library Well_col	i5 well	Barcode_96_Well_Position	i5 name	i5 plate	i5 well_row	i5 well_col	barcode_set_id	barcode_id	MiniPico Library DNA Concentration	MiniPico Library Concentration	sample sheet Sample_ID	MiniPico Pooled Volume	Well_description
 01LJ01593.V8	10	TestProjA_10002_1_2_3_4_2to1dilution	20240911	1	TestProjA	TestProjA_10002	TestProjA_10002_Plate_1_2to1dilution	TestProjA_Plate_1	A	93522	359190605	A19	70	A10					False	0.6333	3.1665	6.333	3.1665	True	0.0999999999999999	A19	False	2367.5	2632.5	7.496688750000001		A	19	A1	A1	C501	TellSeq_Barcode_Plate_1_LN2409001_EXP052026	A	1	col19to24	C501	0.635	1.924242424242424	01LJ01593_V8	1979.1666666666667	TestProjA_10002_Plate_1_2to1dilution.01LJ01593.V8.A19
 01LJ01300.V5	10	TestProjA_10002_1_2_3_4_10to1dilution	20240911	3	TestProjA	TestProjA_10002	TestProjA_10002_Plate_3_10to1dilution	TestProjA_Plate_3	A	93654	359138235	B19	70	A10					False	6.5469	32.7345	65.469	6.5469	True	0.1	B19	False	1145.0	3855.0	7.4962005		B	19	B1	A2	C509	TellSeq_Barcode_Plate_1_LN2409001_EXP052026	B	1	col19to24	C509	5.113	15.493939393939396	01LJ01300_V5	1979.1666666666667	TestProjA_10002_Plate_3_10to1dilution.01LJ01300.V5.B19
-01LJ02603.V8	10	TestProjA_10002_1_2_3_4_2to1dilution	20240911	1	TestProjA	TestProjA_10002	TestProjA_10002_Plate_1_2to1dilution	TestProjA_Plate_1	B	93522	359190229	C19	70	B10					False	1.3558	6.779	13.558	6.779	True	0.0999999999999999	C19	False	1107.5	3892.5	7.5077425		C	19	C1	B1	C502	TellSeq_Barcode_Plate_1_LN2409001_EXP052026	C	1	col19to24	C502	0.677	2.051515151515152	01LJ02603_V8	1979.1666666666667	TestProjA_10002_Plate_1_2to1dilution.01LJ02603.V8.C19
+01LJ02603.V8	10	TestProjA_10002_1_2_3_4_2to1dilution	20240911	1	TestProjA	TestProjA_10002	TestProjA_10002_Plate_1_2to1dilution	TestProjA_Plate_1	B	93522	359190229	C19	70	B10					False	1.3558	6.779	13.558	6.779	True	0.0999999999999999	C19	False	1107.5	3892.5	7.5077425		C	19	C1	B1	C502	TellSeq_Barcode_Plate_1_LN2409001_EXP052026	C	1	col19to24	C502	0.677	2.0515151515151517	01LJ02603_V8	1979.1666666666667	TestProjA_10002_Plate_1_2to1dilution.01LJ02603.V8.C19
```

Comment:
These are interesting differences--small (probably too small to cause problems) but noticeable! Have you thought about what might have caused them?